final/subgraph-locations/locations.graphql

```diff
@@ -1,11 +1,15 @@
+extend schema
```

Comment:
Does the reviews subgraph need this `extend schema` bit too?

Reply:
great catch! you were faster than i was 😆 it should be removed now @meganesu

Reply:
As far as I know, it does need to include it!

Reply:
thank you for the catch, updated @meganesu !